backend/app/src/main/java/com/moh/phlat/backend/controller/ProcessDataController.java

```diff
@@ -390,8 +394,10 @@ public ResponseEntity<ResponseMessage> updateProcessDataById(@RequestBody @Valid
 
 	@PreAuthorize("hasAnyRole(@roleService.getAllRoles())")
 	@GetMapping("/getformfields/header")
-	public String getAllHeader() {
-		return dbUtilityService.getVariablesByTableNameSortedById("PROCESS_DATA");
+	public ResponseEntity<ResponseMessage> getAllHeader() {
```

Comment:
method name also can be changed accordingly to reflect the meaning. I suggest getDisplayColumnNames

Reply:
I made the change as per your suggestion.